### DISCUSSION TITLE
暂停时隐藏控制条

### DISCUSSION THREAD

**Discussion by mingyEx** [the1812/Bilibili-Evolved#4502] [功能建议 / Ideas]
就像youtube的 ** hyde — hide the YouTube video player controls插件 **一样,这样截图的时候比较容易，万一错过也不再需要等那一秒

**Comment by Geeyun-JY3** [comment]
如果只是要截图完全可以试试本脚本的[`启用视频截图`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E5%90%AF%E7%94%A8%E8%A7%86%E9%A2%91%E6%88%AA%E5%9B%BE)。

**Reply by mingyEx** [reply]
感谢回复，我现在才发现b站的播放器，在暂停之后，控制条按钮不会一直存在，所以我只需要稍微等一下就可以了。
之前我以为它会跟youtube一样暂停后四角是渐暗且控制条一直存在的

**Comment by Tinhone** [comment]
需要微调视频时间的话也可以试试本脚本的[`启用逐帧调整`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E5%90%AF%E7%94%A8%E9%80%90%E5%B8%A7%E8%B0%83%E6%95%B4)。
